Analysis/EventFiltering/filterTables.h

```diff
@@ -17,19 +17,29 @@ namespace o2::aod
 {
 namespace filtering
 {
-DECLARE_SOA_COLUMN(H2, hasH2, bool);
-DECLARE_SOA_COLUMN(H3, hasH3, bool);
-DECLARE_SOA_COLUMN(He3, hasHe3, bool);
-DECLARE_SOA_COLUMN(He4, hasHe4, bool);
+DECLARE_SOA_COLUMN(H2, hasH2, bool);   //!
+DECLARE_SOA_COLUMN(H3, hasH3, bool);   //!
+DECLARE_SOA_COLUMN(He3, hasHe3, bool); //!
+DECLARE_SOA_COLUMN(He4, hasHe4, bool); //!
+
+// diffraction
+DECLARE_SOA_COLUMN(DG, hasDG, bool); //! Double Gap events, DG
 
 } // namespace filtering
 
-DECLARE_SOA_TABLE(NucleiFilters, "AOD", "Nuclei Filters", filtering::H2, filtering::H3, filtering::He3, filtering::He4);
+DECLARE_SOA_TABLE(NucleiFilters, "AOD", "Nuclei Filters", //!
+                  filtering::H2, filtering::H3, filtering::He3, filtering::He4);
 
 constexpr std::array<char[32], 1> AvailableFilters{"NucleiFilters"};
 constexpr std::array<char[16], 1> FilterDescriptions{"Nuclei Filters"};
```

Comment:
You should update the AvailableFilters and FilterDescriptions with a meaningful name for the Diffractive Filter (i.e. "DiffractiveFilters" and "Diffractive Filters"). These names are important because they will be used in the json file to switch on/off the different software triggers.

Reply:
Ok, I was not aware of this.